.github/workflows/ghstack_land.yml

```diff
@@ -0,0 +1,28 @@
+name: Propose to merge ghstack orig PRs to main
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - 'gh/*/[0-9]+/base'
+jobs:
+  ghstack_merge_to_main:
+    name: Try to create a PR with ghstack /orig branch
+    runs-on: ubuntu-22.04
+    environment: cherry-pick-bot
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: '0'
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    - name: Try to merge PR to main
+      run: |
+        pip install pygithub
+
+        PR_NUMBER=$(echo "$GITHUB_REF" | grep -oE '[0-9]+')
+
+        python .github/scripts/propose_ghstack_orig_pr.py --pr $PR_NUMBER --repo pytorch/executorch
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_PYTORCHBOT_CHERRY_PICK_TOKEN }}
```

Comment:
@huydhn @kit1980 Can we accept `gh/*/base` to trigger the cherry pick bot?

Users already needs permission to push to pytorch:gh/*/base, so this won't grant additional permission